bcda/router.go

```diff
@@ -17,8 +19,12 @@ func NewRouter() http.Handler {
 		}
 	})
 	r.Route("/api/v1", func(r chi.Router) {
-		r.Get("/claims/{acoId}", bulkRequest)
-		r.Get("/job/{jobId}", jobStatus)
+		r.With(auth.RequireTokenAuth).Post("/claims", bulkRequest)
+		r.With(auth.RequireTokenAuth).Get("/job/{jobId}", jobStatus)
```

Comment:
Would it make sense for this endpoint to be /jobs/{jobId}? I think plural is the usual recommendation and it would be consistent with /claims